src/module/api/timeline/search/index.ts

```diff
@@ -0,0 +1,50 @@
+import apigents from "@venturemark/apigents";
```

Comment:
I changed the import statement because `import * as apigents* was returning the module with first property "default" but with Update, Timeline, etc as nested entities. Since "default isn't part of the type definition I can't use it to access entities.